hg-fast-export.py

```diff
@@ -122,6 +127,27 @@ def get_author(logmessage,committer,authors):
       return r
   return committer
 
+def generate_git_lfs_pointer(filename):
+  size = 0
+  hasher = hashlib.sha256()
+  with open(filename, "rb") as bfile:
+    buf = bfile.read(1024*1024*1024)
+    while buf:
+      hasher.update(buf)
+      size = size + len(buf)
+      buf = bfile.read(1024*1024*1024)
+  digest = hasher.hexdigest()
+  try:
+    os.makedirs("lfs/objects/%s/%s" % (digest[0:2],digest[2:4]))
+  except:
+    pass
+
```

Comment:
Sorry, I missed the part where they say that git lfs uses sha256.

Still, what is your plan for when the large file is not in the hg repo's cache?